src/main/java/org/grails/plugins/springsession/web/http/HttpSessionSynchronizer.java

```diff
@@ -19,19 +19,8 @@
 @Order(SessionRepositoryFilter.DEFAULT_ORDER + 1)
 public class HttpSessionSynchronizer extends OncePerRequestFilter {
 
-    private final SpringSessionConfigProperties springSessionConfigProperties;
     private Boolean persistMutable;
 
-    /**
-     * Constructor for HttpSessionSynchronizer.
-     *
-     * @param springSessionConfigProperties the properties used to configure the session.
-     */
-    public HttpSessionSynchronizer(SpringSessionConfigProperties springSessionConfigProperties) {
```

Comment:
SpringSessionConfigProperties does not make sense as only one its field is used and only here and then it's replaced in the very next line of bean initialization:
https://github.com/RxLogix/spring-session/blob/6.x-upgrade/src/main/groovy/SpringSessionGrailsPlugin.groovy#L105

Plus making it as constructor arg only complicates plugin use as user now must to define `SpringSessionConfigProperties` bean in the context.